src/clzones.cpp

```diff
@@ -18,53 +18,110 @@
 zone_manager::zone_manager()
 {
     types.emplace( zone_type_id( "NO_AUTO_PICKUP" ),
-                   zone_type( translate_marker( "No Auto Pickup" ) ) );
-    types.emplace( zone_type_id( "NO_NPC_PICKUP" ), zone_type( translate_marker( "No NPC Pickup" ) ) );
-    types.emplace( zone_type_id( "LOOT_UNSORTED" ), zone_type( translate_marker( "Loot: Unsorted" ) ) );
-    types.emplace( zone_type_id( "LOOT_FOOD" ), zone_type( translate_marker( "Loot: Food" ) ) );
-    types.emplace( zone_type_id( "LOOT_PFOOD" ), zone_type( translate_marker_context( "perishable food",
-                   "Loot: P.Food" ) ) );
-    types.emplace( zone_type_id( "LOOT_DRINK" ), zone_type( translate_marker( "Loot: Drink" ) ) );
+                   zone_type( translate_marker( "No Auto Pickup" ),
+                              translate_marker( "PC doesn't Auto Pickup items inside the zone" ) ) );
+    types.emplace( zone_type_id( "NO_NPC_PICKUP" ),
+                   zone_type( translate_marker( "No NPC Pickup" ),
+                              translate_marker( "Friendly NPCs don't pickup items inside the zone" ) ) );
+    types.emplace( zone_type_id( "LOOT_UNSORTED" ),
+                   zone_type( translate_marker( "Loot: Unsorted" ),
+                              translate_marker( "Source of items for Sort out loot Zone action. It can overlap with Loot zones of different types." ) ) );
```

Comment:
Seems strange to me to have so many capital letters in the sentence. Also, I'm not sure if new players would easily understand what this "'sort out loot' zone action" is. Probably better to word it like `Place to drop unsorted loot. You can use "sort out loot" zone-action to sort items inside. ...`